### PR TITLE
Add french café Columbus Café & Co

### LINKED ISSUE #3692: Add french café Columbus Café & Co
`name=Columbus Café & Co`
`amenity=cafe`

Countries : France, Belgium, Maroco, Koweït, Bahreïn, Quatar

Currently 51 / 180 shops in the around : https://overpass-turbo.eu/s/SMc
website: https://www.columbuscafe.com/
wikidata: ~~[Q90406058](https://www.wikidata.org/wiki/Q90406058)~~ [Q2984582](https://www.wikidata.org/wiki/Q2984582)

### PR DESCRIPTION
**Pull request by pyrog**
Add french café Columbus Café & Co
Close #3692

### CODE REVIEW

File: brands/amenity/cafe.json [hunk 1]
```diff
@@ -469,6 +469,24 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Columbus Café & Co": {
+    "countryCodes": ["fr"],
```
[the remark this answers is not in the document]
Comment (pyrog) [reply brands/amenity/cafe.json:473]: Yes. But how to specify different website : with different wikidata ?
For belgium it's the french website so we could add "be" at least.

[the remark this answers is not in the document]
Comment (pyrog) [reply brands/amenity/cafe.json]: done

**Comment by bhousel** [comment]
Please remove the redundant matchNames.

**Comment by bhousel** [comment]
You removed the non-redundant ones, and left the redundant one :confused:

**Comment by bhousel** [comment]
Oh I'm sorry @pyrog, how you had it was correct and I didn't look closely enough... 
(I didn't notice the "colOmbus" vs "colUmbus")

**Comment by bhousel** [comment]
I restored them with f1300182
Sorry again for my crankiness. :disappointed_relieved:

**Pull request merged**